.github/workflows/ios.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         working-directory: ios/GSSDKCustomDemo
     steps:
     - uses: actions/checkout@v2
```

Comment:
Why do we need to run tuist? Why do we have a Tuist Project.swift file in the demos anyways…